content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,806 @@
+kind: Service
+metadata:
+    name: helloweb
+    annotations:
+        service.beta.kubernetes.io/azure-dns-label-name: $AZURE_LOADBALANCER_DNS_LABEL_NAME
+spec:
+    ports:
+    - port: 443
+      protocol: TCP
+      targetPort: 8443
+    selector:
+        app: hello
+        tier: web
+    type: LoadBalancer
+```
+🔗 <a href="service.yaml">`service.yaml`</a>
+
+Create a unique DNS name for the LoadBalancer Service and then apply it:
+```bash
+export AZURE_LOADBALANCER_DNS_LABEL_NAME=lb-$(uuidgen) # ❗ The label must start with a lowercase ASCII letter
+envsubst < service.yaml | kubectl apply -f -
+```
+
+Within 2-3 minutes, a load balancer should have been provisioned with a public IP.
+
+```bash
+kubectl get service helloweb
+```
+
+Sample output
+
+```terminal
+$ kubectl get service helloweb
+NAME       TYPE           CLUSTER-IP   EXTERNAL-IP     PORT(S)         AGE
+helloweb   LoadBalancer   10.0.141.1   20.114.151.62   443:30394/TCP   7m15s
+```
+
+The `EXTERNAL-IP` will be different for you and it may be different each time you re-create the LoadBalancer service,
+but it will have a stable DNS host name associated with it
+because you annotated the Service with `azure-dns-label-name`.
+This stable DNS hostname can be used as an alias for your chosen `$DOMAIN_NAME` by creating a [DNS CNAME record](https://www.cloudflare.com/en-gb/learning/dns/dns-records/dns-cname-record/):
+
+```bash
+az network dns record-set cname set-record \
+    --zone-name $DOMAIN_NAME \
+    --cname $AZURE_LOADBALANCER_DNS_LABEL_NAME.$AZURE_DEFAULTS_LOCATION.cloudapp.azure.com \
+    --record-set-name www
+```
+
+Check that `www.$DOMAIN_NAME` now resolves to the ephemeral public IP address of the load balancer:
+
+```terminal
+$ dig www.$DOMAIN_NAME A
+...
+;; QUESTION SECTION:
+;www.cert-manager-tutorial-22.site. IN  A
+...
+;; ANSWER SECTION:
+www.cert-manager-tutorial-22.site. 3600 IN CNAME lb-ec8776e1-d067-4d4c-8cce-fdf07ce48260.eastus2.cloudapp.azure.com.
+lb-ec8776e1-d067-4d4c-8cce-fdf07ce48260.eastus2.cloudapp.azure.com. 10 IN A 20.122.27.189
+...
+```
+
+If the DNS is correct and the load balancer is working and the hello world web server is running,
+you should now be able to connect to it using curl or using your web browser:
+
+```bash
+curl --insecure -v https://www.$DOMAIN_NAME
+```
+
+You should see that the SSL certificate has the expected DNS names and that it is self-signed:
+
+```terminal
+...
+* Server certificate:
+*  subject: CN=www.cert-manager-tutorial-22.site
+*  start date: Jan  4 15:28:30 2023 GMT
+*  expire date: Apr  4 15:28:30 2023 GMT
+*  issuer: CN=www.cert-manager-tutorial-22.site
+*  SSL certificate verify result: self-signed certificate (18), continuing anyway.
+...
+Hello, world!
+Protocol: HTTP/2.0!
+Hostname: helloweb-55cb4cd887-tjlvh
+```
+
+> ⚠️ We used curl's `--insecure` option because it rejects self-signed certificates by default.
+> Later you will learn how to create a trusted SSL certificate signed by Let's Encrypt.
```

Comment:
suggestion: Do you think this info should come _before_ we use `--insecure`?

Most people should see `--insecure` as a red flag, and I think I'd want it to be crystal clear that it won't _stay_ insecure.